src/api/tenantApi.js

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios'
+const BASE_URL = 'https://dev.elevate-apis.shikshalokam.org'
+
+// Get all tenants
+export const getAllTenants = async () => {
+  const response = await axios.get(`${BASE_URL}/user/v1/tenant/list`, {
+    headers: {
+      'x-auth-token': localStorage.getItem('access_token'),
+      'x-tenant': 'mentoring',
+    },
+  })
+  return response.data.result
+}
+
+// Get tenant by code
+export const getTenantByCode = async (code) => {
+  try {
+    const response = await axios.get(`${BASE_URL}/user/v1/tenant/read/${code}`, {
+      headers: {
+        'x-auth-token': localStorage.getItem('access_token'),
+        'x-tenant': 'mentoring',
+      },
+    })
+    return response.data
+  } catch (error) {
+    console.error('❌ Failed to fetch tenant details:', error?.response?.data || error.message)
```

Comment:
Adding to the previous  comment, the error scenario alos should be handled in one place,